homie-controller/src/lib.rs

```diff
@@ -238,6 +240,7 @@ impl HomieController {
         // our Arc to point to that, so that it is now a unique reference.
         let devices = &mut *self.devices.lock().unwrap();
         let devices = Arc::make_mut(devices);
+        let early_property_values = &mut *self.early_property_values.lock().unwrap();
```

Comment:
Nit: add a blank line before this line, to be clear that the comment above is only referring to the `devices`.